src/main/kotlin/org/rust/ide/inspections/RsAttrWithoutParenInspection.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.inspections
+
+import org.rust.lang.core.psi.RsMetaItem
+import org.rust.lang.core.psi.RsVisitor
+import org.rust.lang.core.psi.ext.name
+import org.rust.lang.utils.RsDiagnostic
+import org.rust.lang.utils.addToHolder
+
+class RsAttrWithoutParenInspection : RsLocalInspectionTool() {
+
+    private val attributesWithParen = setOf(
+        "link",
+        "repr",
+        "derive",
+        "repr",
+        "derive",
+        "cfg",
+        "cfg_attr",
+        "allow",
+        "warn",
+        "forbid",
+        "deny",
+        "proc_macro_derive"
+    )
+
+    override fun buildVisitor(holder: RsProblemsHolder, isOnTheFly: Boolean) = object : RsVisitor() {
+        override fun visitMetaItem(metaItem: RsMetaItem) {
+            val name = metaItem.name ?: return
+            if (name in attributesWithParen && metaItem.metaItemArgs == null) {
```

Comment:
Let's also check that `metaItem.parent` is `RsAttr`. 
Otherwise, we can get false positive erros for custom proc-macros.
For example:
`lib.rs`
```rust
use proc_macro::TokenStream;

#[proc_macro_attribute]
pub fn return_as_is(_attr: TokenStream, item: TokenStream) -> TokenStream {
    item
}
```

`main.rs`
```rust
use crate_name::return_as_is;

#[return_as_is(allow)] // <- there isn't error here
fn foo() {}

fn main() {}
```

Of course, in future we should take into account child attributes of `cfg_attr` but we don't convenient API for it currently